en/sdk/cocos-analytics.md

```diff
@@ -1,18 +1,18 @@
-# Cocos 数据统计（Cocos Analytics）
+# Cocos Statistics (Cocos Analytics)
 
-Cocos 数据统计用于记录玩家的游戏行为，并且在后台提供了数据分析支持。Cocos Creator 能一键集成数据统计的 SDK，目前支持网页预览以及发布到 Web，iOS 和安卓平台。
+**Cocos** data statistics are used to record the player's game behavior and provide data analysis in the background. **Cocos Creator** provides one-click integration to get data statistics into your game. Currently, *web page preview* and *publishing to the web*, *iOS* and *Android* platforms are supported.
 
-打开 **项目设置** 面板，位于`菜单栏 - 项目 - 项目设置`。在 **服务** 一栏可以勾选 **Cocos 数据统计** 的配置项：
+Open the **Project Settings** panel, located in **Menu Bar - Projects - Project Settings**. In the **Services** column, you can check the configuration item of **Cocos Data Statistics**:
```

Comment:
1. Menu Bar -> Projects -> Project Settings, use "->" maybe better than "-"? Sorry, it is my fault that the zh document is not well written.
2. Cocos Data Statistics -> Cocos Analytics